### PR TITLE
Minor updates to onboarding

### PR DESCRIPTION
**Pull request by SheepTester**
# Info

adjustments to onboarding

## Changes

- replaced temporary images on first page
- made leaderboard look better on mobile
- whether onboarding was viewed is now stored on backend

# Type of Change

- [x] Bug Fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Logistics Change (A change to a README, description, or dev workflow setup like
      linting/formatting)
- [ ] Continuous Integration Change (Related to deployment steps or continuous integration
      workflows)
- [ ] Other: (Fill In) 

# Testing

I have tested that my changes fully resolve the linked issue ...

- [x] locally on Desktop.
- [ ] on the live deployment preview on Desktop.
- [ ] on the live deployment preview on Mobile.
- [ ] I have added new Cypress tests that are passing.

# Checklist

- [ ] I have performed a self-review of my own code.
- [ ] I have followed the style guidelines of this project.
- [ ] I have documented any new functions in `/src/lib/*` and commented hard to understand areas
      anywhere else.
- [ ] My changes produce no new warnings.

# Screenshots

image

https://github.com/user-attachments/assets/b8822313-bae6-4aa1-aff6-2e6bf40c6e5f

image

### CODE REVIEW

**Review by newracket — APPROVED** [review]
i don't really know how animations work but it looks good to me

File: src/pages/index.tsx
Comment (alexzhang1618) [inline src/pages/index.tsx]: We can remove these comments now right?

File: src/components/onboarding/Intro/index.tsx
Comment (alexzhang1618) [inline src/components/onboarding/Intro/index.tsx]: nit: Let's make these typed so it's easier to read what number corresponds to what

File: src/components/onboarding/Intro/index.tsx
Comment (alexzhang1618) [inline src/components/onboarding/Intro/index.tsx]: This is fine, but let's try to refactor out what we can to make this more readable if possible. I see animation stuff is getting reused, and a lot of logic is basically the same except switching between mobile and desktop. 

Maybe a helper function that returns the args, then we can just spread them here?
i.e. `getImageArgs(i, size, offsetY, offsetX)`

File: src/components/onboarding/Intro/index.tsx
Comment (alexzhang1618) [inline src/components/onboarding/Intro/index.tsx]: Add some comments here on 1. why we're reversing and 2. what the calculation here does. I think this makes sense to store as constants for readability but it's your call

File: src/components/onboarding/Intro/index.tsx
Comment (alexzhang1618) [inline src/components/onboarding/Intro/index.tsx]: Let's try and make these names as distinctive/descriptive as possible. I think
1. BitByteAllocation
2. BitByteInfoSession
3. BitByteSpeedFriending
4. BonfireMarshmallows
5. BonfireBeach (also I think this is Matthew, not Raymond) 👻 
6. KickoffSideView
7. KickoffCloseUp
8. KickoffFrontView

**Review by alexzhang1618 — APPROVED** [review]
Approving to unblock, some nits:

Let's try and rename the asset files to be as terse as possible. I think it's implicit that this is ACM, and we can probably shorten each file name to be something like
FA23_Allocation, FA24_BitByteInfo_1, etc.

File: src/components/onboarding/Intro/index.tsx
Comment (SheepTester) [reply src/components/onboarding/Intro/index.tsx]: I'm assuming you mean to change this into an object with named properties rather than a tuple

**Pull request merged**
